frontend/src/pages/BlogPage.jsx

```diff
@@ -0,0 +1,167 @@
+                </div>
+
+              <div className="cell medium-6">
+                <div className="grid-y">
+                  <div className="cell small-8">
+                  <h1 className="hero__title">Feminist to the core or a carefully polished brand?</h1>
+                  </div>
+
+                  <div className="grid-x">
+                  {/* Profile Picture */}
+                    <div className="grid-container">
+                      <div className="grid-y">
+                        <div className="cell small-6">
+                          <img src={ProfilePic} className="hero__profile" alt="profile"/>
+                        </div>
+                        </div> 
+                      
+
+                    </div>
+                    {/* Author and Date */}
+
+                    <div className="grid-container">
+                      <div className="grid-y">
+                        <div className="cell medium-4">
+                          <h3>Rashmeet Kaur</h3>
+                        </div>
+                        <div className="cell medium-4">
+                          <p>18 May 2020</p>
+                        </div>
+                      </div>
+                    </div>
+                     {/* Social Media Icons */}
+                    <div className="grid-container">
+                      <div className="grid-x">
+                        <div className="cell medium-4">
+                          <img src={Facebook} className="hero__social__icon" alt="facebook"/>
+                        </div>
+                        <div className="cell medium-4">
+                          <img src={Linkedin}  className="hero__social__icon" alt="linkedln"/>
+                        </div>
+                        <div className="cell medium-4">
+                          <img src={Twitter}  className="hero__social__icon" alt="twitter"/>
+                        </div>
+                      </div>
+                    </div>
+
+
+
+                  
+
+
+                  </div>
+
+                </div>
+               
+              </div>
+
+              
+     
+
+                
+                </div>
+               
+              </div>
+            </div>
+
+            <div className="main">
+              <div className="grid-conatiner fluid">
+                <div className="grid-y">
+                  <div className="cell large-8">
+                  <div className="grid-container">
+                    <p className="main__para__head">
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+                                    incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud 
+                                    exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+                    </p>
+                  </div>
+
+                  <div className="grid-container">
```

Comment:
`grid-container` class should only be used once. You have used it almost everywhere.